src/java/fr/paris/lutece/plugins/search/solr/service/SolrSearchAppConfService.java

```diff
@@ -80,6 +81,10 @@ public static SolrSearchAppConf loadConfiguration( String code )
             {
                 conf.setFilterQuery( referenceItemName );
             }
+            else if ( referenceItemCode.matches( DSKEY_FQ_REGULAR_EXPR ) )
```

Comment:
For consistency, it's better to do it the same way as the addonBeans, ie computing the full prefix and use startsWith instead of a regexp